### PR TITLE
types(e2e): Add types and set default timeout to 30s

### PR DESCRIPTION
**Pull request by CodingDive**
After the CI kept running for 3hrs, I wanted to introduce a default timeout. I think it does probably have one by default which makes me wonder why the tests didn't just fail :thinking:

### CODE REVIEW

File: e2e-tests/codecept.config.ts [hunk 1]
```diff
@@ -40,7 +41,9 @@ exports.config = {
       inject: 'login',
       users: {
         admin: {
-          login: (I) => {
+          login: (I: CodeceptJS.I) => {
+            I.amOnPage('/')
+            I.amOnPage('/')
```
Comment (elbotho) [inline e2e-tests/codecept.config.ts]: is that line doubled on purpose?

Comment (CodingDive) [reply e2e-tests/codecept.config.ts]: Yikes! Great find, thank you :pray: Fixed now

**Review by elbotho — APPROVED** [review]
hard to judge without knowing the default timeout, but it's save enough to try :)

**Comment by CodingDive** [comment]
Global expect timeout should be 5 seconds https://playwright.dev/docs/test-timeouts#introduction

**Pull request merged**